Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -131,6 +131,51 @@ def get_external_ip_address_range_request(
 
         return response
 
+    def get_attack_surface_rule_request(
+        self, attack_surface_rule_id: str, enabled_status: str, priority: str, category: str
+    ) -> Dict[str, Any]:
+        """Get Attack Surface Rule details for an attack surface rule id using the '/get_attack_surface_rules/' endpoint.
+
+        Args:
+            attack_surface_rule_id (str): Coma separated attack surface rule ids.
+
+        Returns:
+            dict: dict containing information about Attack surface rule.
+        """
+        filters = []
+        if attack_surface_rule_id != 'None':
```

Comment:
In addition to moving this logic to _command.  I suggest that we don't convert the variable to a string because it should come in as a string anyways.  In that way instead you can do:

> attack_surface_rule_id = args.get("attack_surface_rule_id")
> if attack_surface_rule_id:
>    <other code>

This is what we are doing other places in the integration and is just cleaner to me.  I realize there are other parts of the code we will need to clean this up.

Reply:
Updated in this commit: 0ad1560d46701be1f51498172d3236454085f87f

Reply:
sorry to be a bother, but I don't see this is as resolved.  What I mentioned was to change likes like:

> if attack_surface_rule_id and attack_surface_rule_id != 'None':

to just

> if attack_surface_rule_id:

Reply:
Apologies. Updated here: b9c6b4ecc3c6feaf60bb37e81edce5f0361794e1

Reply:
looks good, thank you